client/actions/test/editActions.test.js

```diff
@@ -1,27 +1,28 @@
 import * as editActions from '../editActions'
-import Article from '../../models/article.coffee'
+import { clone } from 'lodash'
+import Backbone from 'backbone'
 import { Fixtures } from '@artsy/reaction/dist/Components/Publishing'
```

Comment:
(At some point can refactor all of these paths in a different PR to point to new alias?)

Reply:
Yeah adding an alias like force is on my todo list